### DISCUSSION TITLE
Matrix room URLs

### DISCUSSION THREAD

**Discussion by jfrantzius** [containers/podman#21439] [Q&A]
Hi, 
the page https://podman.io/community only lists https://matrix.to/#/#podman:fedoraproject.org as a room on Matrix, but when looking at Discord, there are two additional channels "podman-desktop" and "podman-desktop-dev" which are bridged to Matrix. 

In case somebody else is looking for them, they are
* https://matrix.to/#/#podman-desktop:fedora.im
* https://matrix.to/#/%23podman-desktop-dev:fedora.im

It would be nice to add them to https://podman.io/community , does somebody know which Git repo that would be, if any?

**Comment by eriksjolund** [comment]
It looks like the list is defined here
https://github.com/containers/podman.io/blob/main/static/data/community.ts
